back/config/tenant_templates/e2etests_template.yml

```diff
@@ -15910,26 +15889,6 @@ models:
     ordering: 2
     enabled: false
     code: domicile
-  - &custom_idea_number_field
```

Comment:
I removed this from the template because it was also broken on master and probably has been for ages. It should contain at least one page which it doesn't